extensions/resteasy-classic/rest-client/config/src/main/java/io/quarkus/restclient/config/RestClientsConfig.java

```diff
@@ -45,4 +45,21 @@
     @ConfigItem
     public Optional<String> multipartPostEncoderMode;
 
+    /**
+     * Level of logging for the client.
+     * <br/>
+     * WARNING: beware of logging sensitive data
+     * <br/>
+     * The possible values are:
+     * <ul>
+     * <li>{@code request-response} - enables logging request and responses, including redirect responses</li>
+     * <li>{@code all} - enables logging requests and responses and lower-level logging</li>
+     * <li>{@code none} - no additional logging</li>
+     * </ul>
+     *
+     * This property is applicable to reactive REST clients only.
+     */
+    @ConfigItem
+    public Optional<String> logLevel;
```

Comment:
Shouldn't this use the `LogLevel` enum?

Reply:
the problem is the `RestClientsConfig` class is shared between rest-client and rest-client-reactive, so doesn't have access to RESTEasy Reactive stuff. It's the same for `multipartPostEncoderMode` above

Reply:
Gotcha, thanks